doc/tools/make_rst.py

```diff
@@ -183,6 +183,7 @@ def parse_class(self, class_root: ET.Element, filepath: str) -> None:
 
         operators = class_root.find("operators")
         if operators is not None:
+            index = 0
```

Comment:
I don't understand why you would use an index here. What does it achieve? In fact, if we're giving each operator its own link, then we don't need any kind of grouping at all. But either way, it doesn't seem to matter, so for consistency with the rest of the methods we can just leave this part as it previously was.

Reply:
Okay, if this is to address your second point, then you should just remove `if i == 0` check for operator descriptions. Then every one of them will have an anchor added.